### PR TITLE
removing a single line of text that isn't needed

### PR DESCRIPTION
**Pull request by duckupus**

### CODE REVIEW

**Comment by duckupus** [comment]
Added a new commit... created tips

File: category/web/tools.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#Website directory searching
```
Comment (Mini-Ware) [inline category/web/tools.md:1]: lol

File: category/crypto/tips.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Different encrypthon methods covered here
+- change of base
+- XOR
+- URL encoding
+
+# base
+there are many types of bases,
```
Comment (Mini-Ware) [inline category/crypto/tips.md:7]: maybe list some types down like hex, decimal?

**Review by Mini-Ware — APPROVED** [review]
Everything seems okay, I shall merge it now

File: category/web/tools.md (hunk 1 above)
Comment (duckupus) [reply category/web/tools.md:1]: 🤣

**Pull request merged**